lib/src/widgets/chat/video/ds_video_player.widget.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+
+import 'package:chewie/chewie.dart';
+import 'package:get/get.dart';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/controllers/ds_video_player.controller.dart';
+
+class DSVideoPlayer extends StatelessWidget {
+  final DSVideoPlayerController controller;
+  final String appBarText;
+
+  /// Video player widget
+
```

Comment:
You should add a blank doc comment here.
```suggestion
///
```

Reply:
Done